### ISSUE TITLE
`secretKeyMultibase` size incorrect

### ISSUE THREAD

**Issue by brianorwhatever** [digitalbazaar/ed25519-multikey#11]
This library creates `secretKeyMultibase` values that are 64 bytes in size whereas the [DI spec](https://www.w3.org/TR/vc-data-integrity/#example-multikey-encoding-of-a-ed25519-secret-key) says that they should be only 32 bytes. I assume the public key is appended or prepended and this is what used to be done in the older signature suites and maybe this has been updated? The `@noble/ed25519` crypto package expects only 32 bytes.

**Comment by dlongley** [comment]
Yeah, there was some debate on how this should be formatted and we ended up on cutting it to 32 bytes. We should update this package and do a major release.

**Comment by dlongley** [comment]
There is already some code that does something around this... so we should investigate to figure out what is needed:

https://github.com/digitalbazaar/ed25519-multikey/blob/main/lib/serialize.js#L12

**Comment by dlongley** [comment]
@brianorwhatever,

There's an option, `canonicalize` that can be set to `true` to force old keys that used 64 bytes to output 32 when exporting, see here:

https://github.com/digitalbazaar/ed25519-multikey/blob/main/lib/serialize.js#L14-L15

The current version doesn't mess with imported keys, exporting them the same way they were imported (it looks like). We could have a new major release that defaults to `canonicalize=true` -- or we could leave things as-is.

**Comment by hello0827** [comment]
Hi all i am new to this repo. My guess is generateKeyPairFromSeed() is not correctly implemented?

https://github.com/digitalbazaar/ed25519-multikey/blob/main/lib/ed25519.js
